pkg/steps/artifacts.go

```diff
@@ -155,6 +155,10 @@ func (n *TestCaseNotifier) SubTests(prefix string) []*junit.TestCase {
 	return tests
 }
 
+type podExecInterface interface {
```

Comment:
super nit: Go idiom is to call this `podExecutor` after what it knows how to do

Reply:
I thought about using that exact name, but it could be confused for something that executes pods, instead of commands in pods, so I named it after kube's PodInterface (which itself is not a great name). Changed to `podCmdExecutor`, hopefully somewhere in the middle.